src/components/Person/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {isMarried
+        ? `My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}`
+        : 'I am not married'
+      }
```

Comment:
It's better to create separate variable before return statement where you can store the result of inner checking (sex === 'm')
```suggestion
      {isMarried
        ? `My ${partner}'s name is ${partnerName}`
        : 'I am not married'
      }
```